guby_backend/rocketchat_utils.py

```diff
@@ -0,0 +1,34 @@
+from rocketchat_API.rocketchat import RocketChat
+import os
+import logging
+import traceback
+
+logger = logging.getLogger(__name__)
+
+proxy_dict = {
+    "http": "http://localhost:3000"
```

Comment:
Will this always be the case?

Reply:
If it is so, might still be better to pick from config like you pick `ADMIN_USERNAME` etc?

Reply:
yes agree, will change that.

Reply:
👍